extensions/wikia/WikiAnswers/templates/WikiAnswers_AnswerBox.php

```diff
@@ -3,37 +3,32 @@
 ?>
 			<div id="answer_level">
 			<div id="answer_box" class="accent">
-			<h1 id="answer_heading"><?php echo wfMsg('answer_this_question')?></h1>
+			<h1 id="answer_heading"><?= wfMessage( 'answer_this_question' )->escaped(); ?></h1>
 			<?php
 			$title = $wgTitle;
 		// check whether current user is blocked (RT #48058)
-		$isUserBlocked = $wgUser->isBlockedFrom($title, false);
-
-		if ($isUserBlocked) {
-			//FIXME port the code from Answer::getBlockedInfo()
-			//echo $this->getBlockedInfo();
-		} else {
+		if ( !$wgUser->isBlocked( true ) ) {
```

Comment:
No need to use `User::isBlockedFrom` here as that one is only really relevant for user talk pages, so this way we skip hook execution.